core/vcpu.c

```diff
@@ -2727,7 +2745,8 @@ static void handle_cpuid_virtual(struct vcpu_t *vcpu, uint32_t a, uint32_t c)
             state->_eax = state->_ebx = state->_ecx = 0;
```

Comment:
As @PatrickvL mentioned, this line will cause succeeding state->_ecx always cleared.

Reply:
Yes, ecx zeroing should be removed.